frontend/src/components/Item/itemPreviewList.cy.tsx

```diff
@@ -0,0 +1,78 @@
+import "material-symbols"
+import "../../styles/global.css"
+import { ItemPreviewList } from "./ItemPreviewList"
+
+describe("ItemPreviewList", () => {
+    describe("should display item preview list correct", () => {
+        beforeEach(() => {
+            cy.mount(<ItemPreviewList name="Smartphone" slug="smartphone" weight={{ value: 100, isCa: false }} heaviestWeight={{ value: 100, isCa: false }} imageUrl="https://via.placeholder.com/96.png" datacy="item-preview-list" />)
+        })
+
+        it("should display item preview list correct", () => {
```

Comment:
```suggestion
        it("should display item preview list", () => {
```

Würde sagen der test checkt nur ob sie angezeigt wird.